lib/fenrir_view/accessibility.rb

```diff
@@ -59,6 +59,10 @@ def best_practices_score
       overview_scores&.best_practices
     end
 
+    def colour_issues_count
+      accessibility_report.audits.public_send('color-contrast')&.details&.items&.length || 0
```

Comment:
why are we using public_send ?

Reply:
Because the property is called `color-contrast` with a hyphen, so we can't call it directly.